data/base-help.yaml

```
@@ -917,3 +917,12 @@ water_holder:
     - carve-bead
   specific_descriptions:
     carve-bead:
+
+card_bags:
+  description: Bags you use to collect and store cards
+  example: red backpack
+  referenced_by:
+    - card-collector
+  specific_descriptions:
+    fresh: Fresh bag contains unsorted, newly collected cards. This is where script gets cards for sorting.
+    duplicates: Duplicates bag(bag2) holds anything that doesn\'t fit in the collector\'s case. This also holds collector\'s case.
```

```suggestion
    card-collector:
```

This section is designed to list the scripts that use the setting, and then the specific way in which it uses the setting. Granted, in this case, your setting is designed solely for a single script. You're welcome to leave this section blank if the general description fully encapsulates the use of the setting.

But in the case of other settings, many scripts use them. So we give a top-level general description, and then this `specific-descriptions` section again lists each script that uses the setting, and then allows you to describe the specific way in which it uses it.

See this settings example for `waggle_sets` in `base-help.yaml`: https://github.com/rpherbig/dr-scripts/blob/1cd4361c3e186bcc8d3892377811e94f83a81f43/data/base-help.yaml#L889